docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/going-deeper/example-requests/index.md

```diff
@@ -1,7 +1,5 @@
 ---
 title: "Example Tracker Requests"
-date: "2022-10-10"
-sidebar_position: 100
 ---
 
 # Example Tracker Requests
```

Comment:
I'd do an infobox here just describing that the trackers will perform these network requests for you but if you want more detail, or to write a new tracker implementation then this page is helpful.